hebi_a-2085-06-parallel-gripper_moveit_config/config/A-2085-06.srdf

```diff
@@ -29,14 +30,7 @@
         <link name="end_effector/r_gripper" />
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="home" group="hebi_arm">
-        <joint name="HEBI/base/X8_9" value="0" />
-        <joint name="HEBI/elbow/X8_9" value="2" />
-        <joint name="HEBI/shoulder/X8_16" value="1" />
-        <joint name="HEBI/wrist1/X5_1" value="2.57" />
-        <joint name="HEBI/wrist2/X5_1" value="-1.57" />
-        <joint name="HEBI/wrist3/X5_1" value="0" />
-    </group_state>
```

Comment:
group_state got removed again (I'll just note this on each .srdf so we have it marked)